### PR TITLE
Refactoring filemanager/execute.php

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.5.x
| Description?  | Clear path and path_thumb.
| Type?         | refacto
| Category?     |  BO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | You should be able to create directory / file, upload, rename and delete.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11457)

### CODE REVIEW

File: admin-dev/filemanager/execute.php [hunk 1]
```diff
@@ -31,12 +33,7 @@
 require_once $language_file;
 
 $base = $current_path;
-
-if (isset($_POST['path'])) {
-    $path = $current_path.str_replace("\0", "", $_POST['path']);
-} else {
-    $path = $current_path;
-}
+$path = isset($_POST['path']) ? ($current_path . $_POST['path']) : $current_path;
```
Comment (Quetzacoalt91) [inline admin-dev/filemanager/execute.php:36]: With your PR, `isset($_POST['path'])` is now always true.

File: admin-dev/filemanager/execute.php [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 if ($_SESSION['verify'] != 'RESPONSIVEfilemanager') {
-    die('forbiden');
+    die('Forbidden');
 }
 include('include/utils.php');
 
-$_POST['path_thumb'] = $thumbs_base_path.$_POST['path_thumb'];
-if (!isset($_POST['path_thumb']) && trim($_POST['path_thumb']) == '') {
+$_POST['path'] = isset($_POST['path']) ? str_replace("\0", '', $_POST['path']) : null;
+$_POST['path_thumb'] = isset($_POST['path_thumb']) ? $thumbs_base_path . str_replace("\0", '', $_POST['path_thumb']) : null;
```
Comment (Quetzacoalt91) [inline admin-dev/filemanager/execute.php:9]: Why choosing `null` and not an empty string?

File: admin-dev/filemanager/execute.php (hunk 1 above)
Comment (PierreRambaud) [reply admin-dev/filemanager/execute.php:36]: There are many require above, I can't trust anything, that's why I keep the isset test :/

File: admin-dev/filemanager/execute.php (hunk 2 above)
Comment (PierreRambaud) [reply admin-dev/filemanager/execute.php:9]: Just because if `$_POST['thisIsAKey']` doesn't exists, it's logically null, but I can change if you want.

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Thank you @PierreRambaud

**Pull request merged**